src/coreclr/vm/peimage.inl

```diff
@@ -32,12 +32,25 @@ inline const SString &PEImage::GetPath()
     return m_path;
 }
 
+#if defined(TARGET_ANDROID)
+inline const HANDLE PEImage::AndroidGetDataStart()
```

Comment:
Can't this be `GetFileHandle()`?

Reply:
I'd rather keep this name, as the pointer isn't really a file handle - it's a pointer to a memory area which may, but doesn't have to, correspond to a file.

Reply:
@AaronRobinsonMSFT is `FileHandle` what we usually call such a thing in runtime? I think it would make sense to use whatever is common in runtime.

Reply:
`HANDLE` used for things that you dispose by calling `CloseHandle`. This is not a handle. This is a pointer.

The type should be `void*` if the external mapping is R/W; or `const void* if it is R/O.

The name can be DataStart or just Data. However, it is a security best practice to always pass a buffer size together with the pointer the buffer, so I would make the function return the size of the payload too.

Reply:
The size is stored alongside the pointer in `m_bundleFileLocation`, and this function is a variation on what's already done in `Bundle` mode.  In the current implementation, the pointer to data is implied to be the start of the bundle file, with offset returned by `PEImage::GetOffset()` and size returned by `PEImage::GetSize()`. The change for this function to return both the pointer and the size is, I believe, another item for a follow-up since the change would be much larger in scope than the current PR. Also, I don't know the code good enough to be comfortable in making this change at this point.

Please keep in mind that this PR is meant to be merely a beginning of a longer process, so changes like what you propose can and should be made in follow-up PRs.